src/Service/InstallationService.php

```diff
@@ -996,6 +1003,9 @@ private function createTemplate(array $templateData): ?Template
 
             unset($templateData['organization']);
             $templateData['organization'] = $organization;
+        } else {
+            echo 'setting default organization';
+            $templateData['organization'] = $this->entityManager->getRepository('App:Organization')->find('a1c8e0b6-2f78-480d-a9fb-9792142f4761');
```

Comment:
Using the reference ipv id looks and feels better here (https://docs.commongateway.nl/organization/default.organization.json), maybe do it in a way like $testDataDefault is used in installationService:
![image](https://github.com/CommonGateway/CoreBundle/assets/3785488/aca4fee4-c5f0-4043-b7a7-f8c78b43de83)